src/test/java/com/iexec/worker/sms/SmsServiceTests.java

```diff
@@ -41,33 +41,51 @@
 
 class SmsServiceTests {
 
-    private static final String CAS_URL = "http://cas";
-    private static final String SESSION_ID = "randomSessionId";
+    private final static String CHAIN_TASK_ID = "chainTaskId";
+    private final static TaskDescription TASK_DESCRIPTION = TaskDescription
+            .builder()
+            .chainTaskId(CHAIN_TASK_ID)
+            .build();
+    private final static String HASH = "hash";
+    private final static WorkerpoolAuthorization WORKERPOOL_AUTHORIZATION = spy(WorkerpoolAuthorization
+            .builder()
+            .chainTaskId(CHAIN_TASK_ID)
+            .build());
+
+    private final static TeeSessionGenerationResponse SESSION = mock(TeeSessionGenerationResponse.class);
+
```

Comment:
Modifiers have a priority order in java.
Prefer `private static final` to avoid code smells.
https://sonarcloud.io/organizations/iexecblockchaincomputing/rules?languages=java&open=java%3AS1124

Reply:
Thanks for this first fix. It was more of a reminder than a request for action right now.
I totally agree to fix this across several pull requests.